ui/src/api.js

```diff
@@ -1064,6 +1087,95 @@ const Api = (req) => {
             });
         },
 
+        duplicatePolicyVersion(
+            domainName,
+            policyName,
+            sourceVersionName,
+            newVersionName,
+            _csrf
+        ) {
+            return new Promise((resolve, reject) => {
+                fetchr.updateOptions({
+                    context: {
+                        _csrf: _csrf,
+                    },
+                });
+                var params = {
+                    domainName,
+                    policyName,
+                    policyOptions: {
+                        fromVersion: sourceVersionName,
+                        version: newVersionName,
+                    },
+                };
+                fetchr
+                    .create('policy-version')
+                    .params(params)
+                    .end((err, data) => {
+                        if (err) {
+                            reject(err);
+                        } else {
+                            resolve(data);
+                        }
+                    });
+            });
+        },
+
+        setActivePolicyVersion(domainName, policyName, version, _csrf) {
+            return new Promise((resolve, reject) => {
+                fetchr.updateOptions({
+                    context: {
+                        _csrf: _csrf,
+                    },
+                });
+                var params = {
+                    domainName,
+                    policyName,
+                    policyOptions: {
+                        version: version,
+                    },
+                };
+                fetchr
+                    .update('policy-version')
+                    .params(params)
+                    .end((err, data) => {
+                        if (err) {
+                            reject(err);
+                        } else {
+                            resolve(data);
+                        }
+                    });
+            });
+        },
+
+        getPolicyVersion(domainName, policyName, version) {
+            // escape zero version
+            let versionEscaped = version;
```

Comment:
Not entirely sure why but if I pass '0' as the version the backend receives it as '{version}'. It might be an rdl rest client issue.
Fixed it by surrounding it with quotes